bundles/binding/org.openhab.binding.fritzboxtr064/src/main/java/org/openhab/binding/fritzboxtr064/internal/Tr064Comm.java

```diff
@@ -179,7 +179,7 @@ public String getTr064Value(ItemConfiguration request) {
 
             String itemCommand = itemConfiguration.getItemCommand();
 
-            if (values.containsKey(itemCommand)) {
+            if (values.containsKey(itemConfiguration)) {
```

Comment:
Explain this change, please.

Reply:
The `values` map is declared as `Map<ItemConfiguration, String>`, so it is clear that any key will need to be of type `ItemConfiguration`. `itemCommand` is declared as a String, so I assume the original code had a typo here. I assumed that the intention was to check whether `itemConfiguration` was present as a key.